afs/afs-core/src/main/java/com/powsybl/afs/AppData.java

```diff
@@ -65,19 +67,21 @@ public AppData(ComputationManager computationManager, Supplier<AppLogger> logger
                 new ServiceLoaderCache<>(FileExtension.class).getServices(),
                 new ServiceLoaderCache<>(ProjectFileExtension.class).getServices(),
                 new ServiceLoaderCache<>(ServiceExtension.class).getServices(),
-                loggerFactory);
+                new LocalTaskMonitor(), loggerFactory);
     }
 
     public AppData(ComputationManager computationManager, List<AppFileSystemProvider> fileSystemProviders,
                    List<FileExtension> fileExtensions, List<ProjectFileExtension> projectFileExtensions,
                    List<ServiceExtension> serviceExtensions) {
-        this(computationManager, fileSystemProviders, fileExtensions, projectFileExtensions, serviceExtensions, NoOpAppLogger::new);
+        this(computationManager, fileSystemProviders, fileExtensions, projectFileExtensions, serviceExtensions,
+                new LocalTaskMonitor(), NoOpAppLogger::new);
```

Comment:
Use LocalTaskMonitor::new to be consistent

Reply:
this is not a lambda